src/components/daydream-controls.js

```diff
@@ -0,0 +1,204 @@
+var registerComponent = require('../core/component').registerComponent;
+var bind = require('../utils/bind');
+var isControllerPresent = require('../utils/tracked-controls').isControllerPresent;
+
+var DAYDREAM_CONTROLLER_MODEL_BASE_URL = 'https://cdn.aframe.io/controllers/google/';
+var DAYDREAM_CONTROLLER_MODEL_OBJ_URL = DAYDREAM_CONTROLLER_MODEL_BASE_URL + 'vr_controller_daydream.obj';
+var DAYDREAM_CONTROLLER_MODEL_OBJ_MTL = DAYDREAM_CONTROLLER_MODEL_BASE_URL + 'vr_controller_daydream.mtl';
+
+var GAMEPAD_ID_PREFIX = 'Daydream Controller';
+
+/**
+ * Vive Controls Component
+ * Interfaces with vive controllers and maps Gamepad events to
+ * common controller buttons: trackpad, trigger, grip, menu and system
+ * It loads a controller model and highlights the pressed buttons
+ */
+module.exports.Component = registerComponent('daydream-controls', {
+  schema: {
+    hand: {default: 'right'}, // This informs the degenerate arm model.
```

Comment:
> <a-entity daydream-controls></a-entity>
should behave as you describe with latest PR.  does it not?

> The arm model puts the hand too close into the body and leaves the other hand floating. 
w.r.t. leaving the other hand floating - that is because the hands are left visible by default, rather than only making visible when controller is detected.  easy exercise left to the reader...

w.r.t. arm model - it was configurable, @dmarcos wanted to hide the settings :-/  

can you adjust the parameters to something that works better for you, that we can try as well?

I had a codepen that was doing laser pointer, but you definitely need an arm model or that feels very broken.